### ISSUE TITLE
Update direction example is broken

### ISSUE THREAD

**Issue by fabianp** [openopt/copt#31]
The example https://storage.googleapis.com/copt/auto_examples/frank_wolfe/plot_vertex_overlap.html#sphx-glr-auto-examples-frank-wolfe-plot-vertex-overlap-py

does not produce the expected output.

@gideonite: do you want to take a look into it? This looks like a good issue to get familiar with the codebase

**Comment by gideonite** [comment]
Yes, interesting. I'll check it out! Sorry, somehow I missed that I was tagged in this issue! Feel free to assign to me.

**Comment by fabianp** [comment]
awesome, assigned

**Comment by GeoffNN** [comment]
Has this been fixed?

For away/pairwise, it would also be interesting to see the results. For pairwise, we may need to store the pair of vertices that are chosen.

**Comment by GeoffNN** [comment]
Shouldn't the overlap go back to 0 every time we change chosen vertex? Or are we plotting the cumulative sum right now?

**Comment by gideonite** [comment]
I tried to fix this in https://github.com/openopt/copt/pull/34. I think it's the cumulative sum, see https://github.com/gideonite/copt/blob/11c0d1b59de0b9da01b3af7bef90a95de8f87d60/examples/frank_wolfe/plot_vertex_overlap.py#L54
